Description: nall/Makefile fixes

* Use the flags from CFLAGS, CPPFLAGS and LDFLAGS. They are set in the rules file.
* Use g++ instead of g++-4.9.

Author: Tobias Hansen <thansen@debian.org>

--- a/nall/GNUmakefile
+++ b/nall/GNUmakefile
@@ -29,19 +29,19 @@
     platform := linux
     delete = rm -f $1
   else ifneq ($(findstring BSD,$(uname)),)
-    platform := bsd
+    platform := linux
     delete = rm -f $1
   else
     $(error unknown platform, please specify manually.)
   endif
 endif
 
-cflags := -x c -std=c99
+cflags := -x c -std=c99 $(CFLAGS) $(CPPFLAGS)
 objcflags := -x objective-c -std=c99
-cppflags := -x c++ -std=c++14
+cppflags := -x c++ -std=c++14 $(CFLAGS) $(CPPFLAGS)
 objcppflags := -x objective-c++ -std=c++14
 flags :=
-link :=
+link := $(LDFLAGS)
 
 # compiler detection
 ifeq ($(compiler),)
@@ -51,7 +51,7 @@
   else ifeq ($(platform),macosx)
     compiler := clang++
   else ifeq ($(platform),linux)
-    compiler := g++-4.9
+    compiler := g++
   else ifeq ($(platform),bsd)
     compiler := g++49
   else
